### DISCUSSION TITLE
S-132 Mounting System?

### DISCUSSION THREAD

**Discussion by GrantMeStrength** [thehighnibble/thehighnibble.github.io#54] [S-132 add-on for IMSAI & Cromemco replicas]
I was wondering about the screw/bolt that's part of the S-132. Is this a mechanism to attach it a little more firmly to the back acrylic panel on the IMSAI/Cromemco? Does it require a newer version of the back sheet with an extra hole? I wanted to check before I try to laser cut or drill a hole myself in the system I have.

**Comment by IMSAI-Oracle** [comment]
From the BOM


  | 1 | M4x10mm Hex Socket ScrewThe M4 socket screw is used to fix the S-132 to the clear acrylic back of the IMSAI80080/Cromemco Z-1. 
       You will need to tap or drill a hole to secure the M4 socket screw.

  | 1 | M4 Hex Nut NylonThe nut can be used as a spacer between the S-132 PCB and the acrylic back of the IMSAI80080/Cromemco Z-1


Lynn M.

**Comment by dmcnaugh** [comment]
The options for this that I can see are:
- drill a smaller hole and tap it for the M4 socket screw - this is the solution I have used
- drill a hole big enough of the M4 socket screw and secure it with another M4 nut - this is an option I know others have used successfully
- drill a suitable sized hole and use one of the knurled fixings (M4) that have to be melted into position usually on the end of a soldering iron - I have not seen this done in acrylic and so don't know how it would turn out
